### PR TITLE
Add SendEvent support to JavaScript plugin runtime

### PR DESCRIPTION
**Pull request by wwitzel3**
Signed-off-by: Wayne Witzel III <wayne@riotousliving.com>

Enables JavaScript plugins to send events via the API to a websocket client.

### CODE REVIEW

**Comment by wwitzel3** [comment]
@scothis this should unblock your redirect issue in the plugin.

File: vendor/github.com/hashicorp/go-plugin/go.mod [hunk 1]
```diff
@@ -1,5 +1,7 @@
 module github.com/hashicorp/go-plugin
 
+go 1.14
```
Comment (scothis) [inline vendor/github.com/hashicorp/go-plugin/go.mod:3]: rm, something in our toolchain is modifying the vendor dir

**Comment by scothis** [comment]
Works in my local test

**Comment by scothis** [comment]
Opened https://github.com/wwitzel3/octant/pull/37 that builds on this PR to handle content path "redirects"

**Comment by wwitzel3** [comment]
@scothis I'm ready to merge this if your local test works and you approve it.

**Pull request merged**